wolfcrypt/src/pwdbased.c

```diff
@@ -449,9 +455,26 @@ int wc_PKCS12_PBKDF_ex(byte* output, const byte* passwd, int passLen,
     for (i = 0; i < (int)pLen; i++)
         P[i] = passwd[i % passLen];
 
+#ifdef WOLFSSL_SMALL_STACK
+    B1 = (mp_int *)XMALLOC(sizeof(*B1), heap, DYNAMIC_TYPE_TMP_BUFFER);
+    if (B1 == NULL) {
+        ret = MEMORY_E;
+        goto out;
+    }
+    i1 = (mp_int *)XMALLOC(sizeof(*i1), heap, DYNAMIC_TYPE_TMP_BUFFER);
+    if (i1 == NULL) {
+        ret = MEMORY_E;
+        goto out;
+    }
+    res = (mp_int *)XMALLOC(sizeof(*i1), heap, DYNAMIC_TYPE_TMP_BUFFER);
+    if (res == NULL) {
```

Comment:
Might be cleaner to just have a single line here with `if (B1 == NULL || i1 == NULL || res == NULL)`

Reply:
done